ajax/libs/vue-functional-calendar/package.json

```
@@ -0,0 +1,63 @@
+{
+  "name": "vue-functional-calendar",
+  "title": "Vue.js Functional Calendar",
+  "version": "2.8.69",
+  "description": "Lightweight, high-performance calendar component based on Vue.js",
+  "keywords": [
+     "vue",
+    "calendar",
+    "component",
+    "vue-component",
+    "functional",
+    "date picker",
+    "date range",
+    "range",
+    "multiple calendars",
+    "multiple",
+    "modal calendar",
+    "modal",
+    "inline",
+    "date",
+    "change month",
+    "change year",
+    "picker",
+    "day",
+    "month",
+    "year",
+    "time",
+    "add",
+    "cli",
+    "command",
+    "keys",
+    "keyword",
+    "keywords",
+    "line",
+    "package",
+    "pkg"
+  ],
+  "homepage": "https://github.com/ManukMinasyan/vue-functional-calendar",
+  "repository": {
+     "type": "git",
```

```suggestion
    "type": "git",
```
2 space indentation, please.